barretenberg/cpp/src/barretenberg/stdlib/primitives/curves/bn254.hpp

```diff
@@ -12,6 +12,7 @@ template <typename CircuitBuilder> struct bn254 {
     // classes are instantiated with "native" curve types. Eventually, the verifier classes will be instantiated only
     // with stdlib types, and "native" verification will be acheived via a simulated builder.
     static constexpr bool is_stdlib_type = true;
+    using NativeCurve = curve::BN254;
```

Comment:
leave a TODO here and below?

Reply:
I think these are actually just fine/logical. When you and I were debugging we had to add these to the native types which didnt make sense  but I was able to get rid of those